src/data/fintype/card.lean

```diff
@@ -242,10 +242,11 @@ lemma fintype.sum_pow_mul_eq_add_pow
 finset.sum_pow_mul_eq_add_pow _ _ _
 
 lemma fin.sum_pow_mul_eq_add_pow {n : ℕ} {R : Type*} [comm_semiring R] (a b : R) :
-  ∑ s : finset (fin n), a ^ s.card * b ^ (n - s.card) =
-  (a + b) ^ n :=
+  ∑ s : finset (fin n), a ^ s.card * b ^ (n - s.card) = (a + b) ^ n :=
 by simpa using fintype.sum_pow_mul_eq_add_pow (fin n) a b
 
+lemma fin.prod_const [comm_monoid α] (n : ℕ) (x : α) : ∏ i : fin n, x = x ^ n := by simp
```

Comment:
This should be `@[to_additive]` right?

Reply:
`to_additive` can't deal with `pow` by itself, so we'll need a manual `sum_const` lemma.

Reply:
We still should connect these two lemmas with `@[to_additive]` after proving them both, right?

Reply:
No, `to_additive` won't be able to use this because it'll fail to match the types of RHS.